packages/core/test/unit/query-generator/remove-index-query.test.ts

```diff
@@ -13,6 +13,7 @@ describe('QueryGenerator#removeIndexQuery', () => {
       ibmi: `BEGIN DROP INDEX "user_foo_bar"; COMMIT; END`,
       db2: `DROP INDEX "user_foo_bar"`,
       postgres: `DROP INDEX "public"."user_foo_bar"`,
+      cockroachdb: `DROP INDEX "public"."user_foo_bar"`,
```

Comment:
It seems that this is the same as postgres, so you can use `'postgres cockroachdb'` in this file